api4/cloud.go

```diff
@@ -411,6 +412,22 @@ func handleCWSWebhook(c *Context, w http.ResponseWriter, r *http.Request) {
 			c.Err = appErr
 			return
 		}
+	case model.EventTypeTrialWillEnd:
+		user, appErr := c.App.GetUserByEmail(event.CloudWorkspaceOwner.UserEmail)
+		if appErr != nil {
+			c.Err = model.NewAppError("Api4.handleCWSWebhook", appErr.Id, nil, appErr.Error(), appErr.StatusCode)
+			return
+		}
```

Comment:
I believe we want to send the email to all admins of the workspace, not just the workspace owner. You can find some examples of sending to all admins for the delinquent payment emails